src/views/Expenses.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import AddExpensesForm from '../components/AddExpensesForm';
+import ExpensesList from '../components/ExpensesList';
+
+const Expenses = () => {
```

Comment:
this class mixes two responsibilities:
1. it displays a list of expenses
2. it interfaces with the api

how do we want to deal with this? should we borrow inspiration from something like redux?

Reply:
Are these responsibilities not split between two components, `Expenses` and `ExpensesList` ?

Reply:
Hey @motine just mentioning you here in case you missed my question above.

Reply:
indeed, i did not see the question.

mmm, I was thinking about a pure "i interface with the API" object which has clearly defined methods. it does not deal with state management at all, it only deals with tokens, API calls. this component right here also does displaying and stuff. I am not sure however how much clearer it would be to separate the data (acquisition) from its view...